frontend/src/app/globals.css

```
@@ -376,207 +646,182 @@ a {
+.question-table tbody tr:last-child td {
+  border-bottom: none;
 }
 
-/* ---- Tag chips in question table ---- */
-
-.question-table-tags {
-  min-width: 160px;
+.question-table-text {
+  min-width: 180px;
+  max-width: 360px;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)